### PR TITLE
Dogo news CSS adjustments #2072

### PR DESCRIPTION
**Pull request by lenniezelk**

### CODE REVIEW

**Comment by moollaza** [comment]
@lenniezelk thanks a lot for contributing! There's actually an ongoing PR for this with some discussion in #2073. Would you be interested in updating your PR to follow the discussion there?

Also, for the future, please add something to your PR description. In general it's best to have a message like "Fixes #2072" as this will cause GitHub to close the mentioned issue when we merge the PR.

I'm not sure that mentioning the PR number in a commit has the same effect?

File: share/spice/dogo_news/dogo_news.css [hunk 1]
```diff
@@ -1,3 +1,18 @@
 .zci--dogo_news .tile--dogo_news .tile__foot {
     padding-bottom: 0.2em;
-}
\ No newline at end of file
+}
+
+#zci-dogo_news .tile__title
+{
```
Comment (moollaza) [inline share/spice/dogo_news/dogo_news.css]: Let's keep with our existing CSS style -- please put the braces at the end of the previous line

File: share/spice/dogo_news/dogo_news.css [hunk 2]
```diff
@@ -1,3 +1,18 @@
 .zci--dogo_news .tile--dogo_news .tile__foot {
     padding-bottom: 0.2em;
-}
\ No newline at end of file
+}
+
+#zci-dogo_news .tile__title
```
Comment (moollaza) [inline share/spice/dogo_news/dogo_news.css]: These should target the `.zci--dogo_news` class, like we've done above

**Comment by lenniezelk** [comment]
@moollaza i was unaware of #2073. I think it would be best to update my PR to follow that discussion.

**Comment by moollaza** [comment]
>@moollaza i was unaware of #2073. I think it would be best to update my PR to follow that discussion.

No problem! Please go ahead and update yours :+1:

**Comment by moollaza** [comment]
@lenniezelk Ping! I'm just checking in to see if you're still interested in finishing this one up?

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by moollaza** [comment]
I'm closing this one for now, but if you'd like to finish this up, we'll gladly open it back up for you!

**Pull request closed without merging**